kili/helpers.py

```diff
@@ -215,18 +216,34 @@ def fragment_builder(fields, type_of_fields):
     return fragment
 
 
-def deprecate(msg, _type=DeprecationWarning):
+def deprecate(
+        msg: Optional[str] = None,
+        removed_in: Optional[str] = None,
+        _type=DeprecationWarning):
     """
```

Comment:
You should add a `use` to describe the case where you can use it, better than any description